composer/core/data_spec.py

```diff
@@ -167,9 +168,9 @@ class DataSpec:
         get_num_tokens_in_batch ((Batch) -> int, optional): Function that is called by the :class:`.Trainer` to
             get the number of tokens in the provided batch.
 
-            By default, it returns 0, meaning that number of tokens processed will not be tracked as a part of the
-            training progress tracking. This function must be specified to track the number of tokens processed during
-            training.
+            By default, it checks for HuggingFace-style dictionary batches with ``input_ids``, and then checks ``dataset.max_seq_len``, and then returns 0
```

Comment:
Can we add some kind of comment related to padding assumptions?

Reply:
added